### PR TITLE
Added `sAddArray` to `Redis` stub

### PR DESCRIPTION
**Pull request by boesing**

### CODE REVIEW

File: Redis.php [hunk 1]
```diff
@@ -3089,4 +3105,4 @@ public function _target($key) {}
      * Use this function when a new node is added and keys need to be rehashed.
      */
     public function _rehash() {}
-}
```
Comment (neuro159) [inline Redis.php]: please avoid unnecessary changes

Comment (boesing) [inline Redis.php]: Sorry, its an IDE thing. 
However, as of `PSR-2`, there MUST be a UNIX LF instead of Windows `CRLF` at the end of a file.
